src/node/NodeDish.mjs

```diff
@@ -0,0 +1,83 @@
+/**
+ * @author d98762625 [d98762625@gmail.com]
+ * @copyright Crown Copyright 2018
+ * @license Apache-2.0
+ */
+
+import util from "util";
+import Dish from "../core/Dish";
+
+/**
+ * Subclass of Dish where `get` and `_translate` are synchronous.
```

Comment:
Might want to consider rewording this as these methods are no longer redefined in this subclass.